python/benchmark/databricks/gpu_etl_cluster_spec.sh

```diff
@@ -0,0 +1,65 @@
+# needed for bm script arguments
+cat <<EOF
+{
+    "num_workers": $num_gpus,
+    "cluster_name": "$cluster_name",
+    "spark_version": "${db_version}.x-gpu-ml-scala2.12",
+    "spark_conf": {
+        "spark.task.resource.gpu.amount": "0.25",
+        "spark.task.cpus": "1",
+        "spark.databricks.delta.preview.enabled": "true",
+        "spark.python.worker.reuse": "true",
+        "spark.executorEnv.PYTHONPATH": "/databricks/jars/rapids-4-spark_2.12-24.06.1.jar:/databricks/spark/python",
+        "spark.sql.files.minPartitionNum": "2",
+        "spark.sql.execution.arrow.maxRecordsPerBatch": "10000",
+        "spark.executor.cores": "8",
+        "spark.executor.memory": "5g",
+        "spark.rapids.memory.gpu.minAllocFraction": "0.0001",
+        "spark.plugins": "com.nvidia.spark.SQLPlugin",
+        "spark.locality.wait": "0s",
+        "spark.sql.cache.serializer": "com.nvidia.spark.ParquetCachedBatchSerializer",
+        "spark.rapids.memory.gpu.pooling.enabled": "false",
+        "spark.rapids.sql.explain": "ALL",
+        "spark.sql.execution.sortBeforeRepartition": "false",
+        "spark.rapids.sql.python.gpu.enabled": "true",
+        "spark.rapids.memory.pinnedPool.size": "2G",
+        "spark.python.daemon.module": "rapids.daemon_databricks",
+        "spark.rapids.sql.batchSizeBytes": "512m",
+        "spark.sql.adaptive.enabled": "false",
+        "spark.sql.execution.arrow.pyspark.enabled": "true",
+        "spark.sql.files.maxPartitionBytes": "2000000000000",
+        "spark.databricks.delta.optimizeWrite.enabled": "false",
+        "spark.rapids.sql.concurrentGpuTasks": "2"
```

Comment:
While we keep this here, do we need to remove it from gpu_cluster_spec.sh (the one without GPU ETL)?

Reply:
Yes.  Will delete there.  Good catch.